hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java

```diff
@@ -535,12 +537,14 @@ public Pair<Boolean, Duration> commitSingleFileByRename(
         throw new PathIOException(qualifiedSrcPath.toString(), "cannot rename object onto self");
       }
 
+      boolean isNamespaceEnabled = abfsStore.getIsNamespaceEnabled(tracingContext);
```

Comment:
Can be added as a part of constructor or initialize method. Code is repeated

Reply:
This needs the tracingContext to be passed as a parameter, which is not meaningful to be created in the initialize method, as it needs params like operation type also. Hence, have kept in the individual methods itself, to use up the tracing context actually created in the API methods.
Would it be better to create a dummy tracing context then?

Reply:
https://github.com/ABFSDriver/AbfsHadoop/blob/d7f81c994178efa2bc70c8c73d20bb8786ef6ea3/hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java#L196 -> you can take reference from this code, have done something similar